apollo-federation/src/link/cost_spec_definition.rs

```diff
@@ -0,0 +1,202 @@
+use std::collections::HashMap;
+
+use apollo_compiler::ast::Argument;
+use apollo_compiler::ast::Directive;
+use apollo_compiler::name;
+use apollo_compiler::schema::Component;
+use apollo_compiler::schema::EnumType;
+use apollo_compiler::schema::ObjectType;
+use apollo_compiler::schema::ScalarType;
+use apollo_compiler::Name;
+use apollo_compiler::Node;
+use lazy_static::lazy_static;
+
+use crate::error::FederationError;
+use crate::link::spec::Identity;
+use crate::link::spec::Url;
+use crate::link::spec::Version;
+use crate::link::spec_definition::SpecDefinition;
+use crate::link::spec_definition::SpecDefinitions;
+use crate::schema::position::EnumTypeDefinitionPosition;
+use crate::schema::position::ObjectTypeDefinitionPosition;
+use crate::schema::position::ScalarTypeDefinitionPosition;
+use crate::schema::FederationSchema;
+
+pub(crate) const COST_DIRECTIVE_NAME_IN_SPEC: Name = name!("cost");
+pub(crate) const COST_DIRECTIVE_NAME_DEFAULT: Name = name!("federation__cost");
+
+pub(crate) const LIST_SIZE_DIRECTIVE_NAME_IN_SPEC: Name = name!("listSize");
+pub(crate) const LIST_SIZE_DIRECTIVE_NAME_DEFAULT: Name = name!("federation__listSize");
```

Comment:
This means we have to make sure we merge in composition 2.9 (and update _our_ composition to 2.10) before the next alpha release